cross/flac/patches/make.in.patch

```diff
@@ -0,0 +1,29 @@
+--- Makefile.in.orig	2007-09-16 22:04:57.000000000 +0200
++++ Makefile.in	2015-03-16 13:07:03.773875616 +0100
+@@ -234,7 +234,7 @@
+ 
+ AUTOMAKE_OPTIONS = foreign 1.7
+ 
+-SUBDIRS = doc include m4 man src examples test build obj
++SUBDIRS = include m4 src build obj
+ 
+ DISTCLEANFILES = libtool-disable-static
+ 
+@@ -290,7 +290,7 @@
+ $(srcdir)/configure: @MAINTAINER_MODE_TRUE@ $(srcdir)/configure.in $(ACLOCAL_M4) $(CONFIGURE_DEPENDENCIES)
+ 	cd $(srcdir) && $(AUTOCONF)
+ 
+-$(ACLOCAL_M4): @MAINTAINER_MODE_TRUE@ configure.in 
++$(ACLOCAL_M4): @MAINTAINER_MODE_TRUE@ configure.in
```

Comment:
Can I propose to revisit this particular comment? It would be best to keep patches limited to the actual content needed. Same with the one below.